### DISCUSSION TITLE
讓AI學會發自內心的笑，需要幾步？

### DISCUSSION THREAD

**Discussion by Kotori0501** [1101-GS4719-C/Python-Community-GS4719-C#812] [General]
依照目前AI現有的技術是無法像人類一樣的表達情緒，若在未來成功處理了，或許可說明AI具有感覺、思考的能力以及難以預料未來的存在和特定職業的人是否會真正的被AI取代。

https://tw.news.yahoo.com/%E8%AE%93ai%E5%AD%B8%E6%9C%83%E7%99%BC%E8%87%AA%E5%85%A7%E5%BF%83%E7%9A%84%E7%AC%91%EF%BC%8C%E9%9C%80%E8%A6%81%E5%B9%BE%E6%AD%A5%EF%BC%9F-081854852.html

**Comment by ghost** [comment]
是否能做出有感情的Ai自然是一大難題，但更困難的是如何證明他的感情是發自內心，而不大數據訓練下，為了符合該狀況的該有的情感，而做出的合理情感。

**Comment by moonnight3503** [comment]
如果未來真的有能表現情感的機器人，著實還蠻可怕的，像許多科幻電影那樣，生活在周遭身邊的人都不知道是機器人還是真人。

**Comment by evanchou1204** [comment]
感覺如果要做出有人類共情、情感的AI機器人，這個方向已經從原本實用主義轉變成有點偏向人文藝術了，很有趣。

**Comment by bonny123865** [comment]
我覺得DALL-E做出來的插畫真的好可愛，有船長貓貓欸 !

**Comment by Doreen331** [comment]
雖然以目前的技術來說，要讓AI「發自內心的笑」很困難，若AI有自己的情緒反應，那就差不多代表他有自己的意識，但若這項技術真的發展成功，那麼人類對於AI最大的恐懼差不多也要成真了

**Comment by yuxinlin93** [comment]
如果AI能發自內心的笑會是很大的突破，但好像也會有點可怕，感覺會發生很多像電影裡面會發生的事，不過就像樓上很多人說的一樣，該怎麼知道他是不是"發自內心"的微笑會是一個很大的困難點

**Comment by a45320614** [comment]
科幻片中的情節離我們的生活越來越近了，未來如果LGBTQ多了一個R(Robot)我也不太意外了

**Comment by ru0372572** [comment]
藝術創作類的作品中，那些令人眼睛為之一亮的，不外乎是那些跳脫框架，在平常的地方做了不一樣處理的創意點子！

**Comment by Pietra1226** [comment]
其實AI人工智能已經很多都通過圖靈測試，表示人類已經有一定程度無法區分真人與人工智慧，感覺再過不久就會有讓人混淆的人工智慧出現。

**Comment by CPCpyt** [comment]
我覺得在完全有可能實現，
雖然機器人表達情感還是很抽象，不過說不定在日後真能夠實現，感覺世界將迎來下一個奇點。

**Comment by ruthchen1020** [comment]
我覺得現在的AI對生命的理解已經比人類還深了，畢竟他讀了全世界人類的訊息。
但可能還沒有到有"意志"的程度，因為人類目前還沒有賦予他可以做對人類有影響的事情。

**Comment by StevenWanglolz** [comment]
如果AI也能有情緒的話，那好像可以完全取代人類了ㄟ，我們會無法分辨是在和人類還是電腦對話

**Comment by Evan0979** [comment]
笑得7 80%像人的話,會很像恐怖片超可怕哈哈哈

**Comment by peggy0329** [comment]
以個人的觀點，我還是比較不希望機器人可以發展到有情緒或是個人表達力
更何況是「發自內心」的情緒，是否有一天他真的有意識了
那人類就會被取代了...

**Comment by dinosaur1009** [comment]
我覺得AI有情緒是一件要仔細思考的議題，因為這感覺也代表著，會危及到人類的生活.....雖然不可否認AI的發展是個趨勢

**Comment by William08290829** [comment]
AI並沒有真正的情感或自我意識，所以「發自內心」這種概念對AI來說應該不適用。但可以通過優化算法和模型，提高AI在處理任務時的效能和準確性，使其看起來更具智慧和反應更迅速，這可能讓人感受到一種「發自內心」的表現。

**Comment by wanderer657** [comment]
我覺得在未來如果AI能夠自行建立模型、設計同類、增加資料數據庫，甚至是擁有靈魂，那跟人類將無限接近，但人類的存在並非毫無意義，因為我們是莫大的變數之一，在尺度更大的時間內，我認為機械還無法駕馭人類這一變數。

**Comment by benbebobo149** [comment]
這個問題可能就像不能說的秘密裡的男主角問:「從我這邊到你那邊需要幾步?」， 我認為可能要先定義甚麼是自主地笑，是寫給程式說笑是必續是嘴角上揚，然後他推理出這個時刻必須有笑的動作，進而有笑這個行為。那請問這是你所定義的自主的笑嗎?

**Comment by lyykey** [comment]
從我這邊到你那邊一共五步
